docs/slackInvite.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Slack invite
+---
+
+<p>Many bosun devs and users are usually availible on our slack channel. If you have any questions or issues this is the best place to 
+get help. Please fill out this form to automatically receive an invitation to join us:</p>
+
+	<div class="form-group">
+		<label for="email">Email address</label>
+		<input type="email" class="form-control" id="email" placeholder="Enter email">
+	</div>
+	<div class="form-group">
+		<label for="fname">First Name</label>
```

Comment:
It is the slack api that reuires first/last. Here is my code running on hookscript if you are curious:

https://gist.github.com/captncraig/d795ffa7990b09b596ed